### ISSUE TITLE
Async support

### ISSUE THREAD

**Issue by bbougot** [doublej42/BitlyAPI#1]
Great projet! Have you planned async support of GetResponse?

**Comment by doublej42** [comment]
Good idea. I'll see if I can find time to add it. I'm terrible with doing async programming but I need to get better.
